docs/core/entity/time.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Time Entity
+sidebar_label: Time
+---
+
+A `time` is an entity that allows the user to input a time to an integration. Derive entity platforms from [`homeassistant.components.time.TimeEntity`](https://github.com/home-assistant/core/blob/dev/homeassistant/components/time/__init__.py)
+
+## Properties
+
+:::tip
+Properties should always only return information from memory and not do I/O (like network requests). Implement `update()` or `async_update()` to fetch data.
+:::
+
+| Name | Type | Default | Description
+| ---- | ---- | ------- | -----------
+| native_value | time | **Required** | The value of the time.
```

Comment:
The `native_value` is a string representation of the python's `time`? Should it be made clearer in the text that both `native_value` and `set_value` are using that?

Reply:
no, both `native_value` and `set_value` expect a `time` object, `state` gets converted to a string representation automatically and the service schema will convert a string input to a `time` object for the service call automatically